src/services/reviews.js

```diff
@@ -98,49 +158,43 @@ module.exports = class reviewsHelper {
 				resource_id: resourceId,
 				reviewer_id: userId,
 			})
+
 			// Check if the review exists; if not, return a failure response
-			if (!review || !review.id) {
-				return responses.failureResponse({
-					message: 'REVIEW_NOT_FOUND',
-					statusCode: httpStatusCode.bad_request,
-					responseCode: 'CLIENT_ERROR',
-				})
-			}
+			if (!review?.id) throw new Error('REVIEW_NOT_FOUND')
+
+			//check the review is already started
+			if (review.status === common.REVIEW_STATUS_STARTED) throw new Error('REVIEW_ALREADY_STARTED')
```

Comment:
Can u also check if there are any other dependancy with this key? in the existing code?

Reply:
@priyanka-TL